index.html

```diff
@@ -2,9 +2,26 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>learne</title>
+    <link rel="stylesheet" href="app.css">
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/jquery.slick/1.6.0/slick.css"/>
+    <script src="https://use.fontawesome.com/ac8a89b97e.js"></script>
   </head>
+
   <body>
 
+    <div class="headWrapper">
+      <h1><i class="fa fa-graduation-cap" aria-hidden="true"></i>
+        Learne</h1>
```

Comment:
The h1 should be inside the container div, so it can be right aligned with it. Currently it is right floated to the whole screen which looks bad